packages/web/src/components/apps/workspace/preview.tsx

```diff
@@ -6,15 +6,27 @@ type PropsType = {
 };
 
 export function Preview(props: PropsType) {
-  const { url } = usePreview();
+  const { url, status } = usePreview();
 
-  if (url === null) {
-    return;
-  }
+  switch (status) {
+    case "booting":
+    case "connecting":
+      return (
+        <div className={cn("flex justify-center items-center w-full h-full", props.className)}>
+          <span className="text-tertiary-foreground">Booting...</span>
+        </div>
+      );
+    case "running":
```

Comment:
Right now, this is what the loading state looks like. An official design (I couldn't find one for something like this) would be appreciated!

<img width="890" alt="Screenshot 2024-10-11 at 4 44 14 PM" src="https://github.com/user-attachments/assets/fddb9cce-a56f-43b4-bdd1-dff49cbac505">